en/deploy/README.md

```diff
@@ -4,243 +4,301 @@
 
 Until now your website was only available on your computer, now you will learn how to deploy it! Deploying is the process of publishing your application on the Internet so people can finally go and see your app :).
 
-As you learned, a website has to be located on a server. There are a lot of providers, but we will use the one with the simplest deployment process: [Heroku](http://heroku.com/). Heroku is free for small applications that don't have too many visitors, it'll definitely be enough for you now.
+As you learned, a website has to be located on a server. There are a lot of providers, but we will one of the ones with the simplest deployment process: [PythonAnywhere](http://pythonanywhere.com/). PythonAnywhere is free for small applications that don't have too many visitors, it'll definitely be enough for you now.
 
-We will be following this tutorial: https://devcenter.heroku.com/articles/getting-started-with-django, but we pasted it here so it's easier for you.
+The other external service we'll be using is [Github](http://www.github.com), which is a code hosting service.  There are others out there, but almost all programmers have a Github account these days, and now so will you!
 
-## The `requirements.txt` file
+We'll use Github as a stepping stone to transport our code to & from PythonAnywhere.
 
-We need to create a `requirements.txt` file to tell Heroku what Python packages need to be installed on our server.
 
-But first, Heroku needs us to install a few packages. Go to your console with `virtualenv` activated and type this:
+# Git
 
-    (myvenv) $ pip install dj-database-url gunicorn whitenoise
+Git is a "version control system" used by a lot of programmers - software which keeps track of changes to a file or set of files over time so that you can recall specific versions later. 
```

Comment:
can we simplify this: `software which keeps track of changes to a file or set of files over time` to this: `software which keeps track of changes to the files over time`. Just easier to understand :)